maas/plugins/horizon_check.py

```diff
@@ -111,5 +111,8 @@ def main(args):
         parser.add_argument('ip',
                             type=ipaddr.IPv4Address,
                             help='horizon dashboard IP address')
+        parser.add_argument('site_name_regexp',
+                            type=str,
+                            help='Horizon Site Name')
```

Comment:
@mancdaz the default is set here [1] and can be overridden in one of the user variables files.

[1] https://github.com/rcbops/rpc-openstack/blob/d2b5efdafbd52ebc49c975f97d2fd25e4a80d1c6/rpcd/playbooks/roles/rpc_maas/defaults/main.yml#L457-L460